src/components/modals/ScanModal.js

```diff
@@ -149,17 +149,17 @@ const Component = (props: Props) => {
             </View>
             <View style={[styles.peepholeSpace, { minHeight: peepholeSize }]} onLayout={handleLayoutPeepholeSpace} />
             <View style={styles.buttonsContainer}>
-              <TouchableHighlight onPress={handleFlash} underlayColor={theme.secondaryButton}>
-                <View style={styles.iconButtonContainer}>
+              <TouchableHighlight style={styles.iconButtonContainer} onPress={handleFlash} underlayColor={theme.secondaryButton}>
+                <>
```

Comment:
I don't think you need the fragment here - `<TouchableHighlight>` takes multiple children.

Reply:
I get an error if I don't have the fragment. React native throws an error saying expecting children lnegth of one.

Reply:
Ok, then we should leave it in.